ui/src/app/utils/resource-list.tsx

```diff
@@ -163,31 +163,41 @@ export const ResourcesList = fp.flow(withCdrVersions())((props: Props) => {
     const { workspaceResources } = props;
     if (workspaceResources) {
       setTableData(
-        workspaceResources.map((r) => {
-          return {
-            resource: r,
-            menu: renderResourceMenu(r),
-            resourceType: getTypeString(r),
-            resourceName: getDisplayName(r),
-            formattedLastModified: displayDateWithoutHours(
-              r.lastModifiedEpochMillis
-            ),
-            lastModifiedDateAsString: displayDate(r.lastModifiedEpochMillis),
-            cdrVersionName: getCdrVersionName(r),
-            workspaceName: getWorkspace(r)?.name,
-            lastModifiedBy: r.lastModifiedBy,
-          };
-        })
+        fp.flatMap((r) => {
```

Comment:
using flatMap because it allows an empty mapping result, and the final array will contain only the valid rows.  LMK if that is unclear or more comments are needed.